Test/Common/TestConfigurations.xml

```diff
@@ -13,15 +13,15 @@
   <TenantConfiguration>
     <TenantName>ProductionTenant</TenantName>
     <TenantType>Cloud</TenantType>
-    <AccountName>[ACCOUNT NAME HERE]</AccountName>
-    <AccountKey>[ACCOUNT KEY HERE]</AccountKey>
-    <BlobServiceEndpoint>http://[ACCOUNT].blob.core.windows.net</BlobServiceEndpoint>
-    <QueueServiceEndpoint>http://[ACCOUNT].queue.core.windows.net</QueueServiceEndpoint>
-    <TableServiceEndpoint>http://[ACCOUNT].table.core.windows.net</TableServiceEndpoint>
-    <FileServiceEndpoint>http://[ACCOUNT].file.core.windows.net</FileServiceEndpoint>
-    <BlobServiceSecondaryEndpoint>http://[ACCOUNT]-secondary.blob.core.windows.net</BlobServiceSecondaryEndpoint>
-    <QueueServiceSecondaryEndpoint>http://[ACCOUNT]-secondary.queue.core.windows.net</QueueServiceSecondaryEndpoint>
-    <TableServiceSecondaryEndpoint>http://[ACCOUNT]-secondary.table.core.windows.net</TableServiceSecondaryEndpoint>
+    <AccountName>wassdktesting</AccountName>
+    <AccountKey>DVWWqT1/0ji/uIOL6yKINT0o8yDByUA2H15VfdJqzWDzHdUgxQ4mZdB2J/B0gXJ9LXeaeX0ZTxcQEAcd3nMkpA==</AccountKey>
+    <BlobServiceEndpoint>http://wassdktesting.blob.core.windows.net</BlobServiceEndpoint>
```

Comment:
You need to remove this file from the PR immediately! You have leaked your testing account creds